benchmarks/report.py

```diff
@@ -0,0 +1,6 @@
+from statistics import mean
+
+def create_report(per_epoch_durations, batch_durations, total_duration):
+    print(f"Total duration is {total_duration}")
+    print(f"Per epoch duration {mean(per_epoch_durations)}")
+    print(f"Per batch duration {mean(batch_durations)}")
```

Comment:
nit: Showing percentile stats (e.g. 1%, 25%, 50%, 75%, 99%) may be more helpful.

Reply:
Revamping this to include a MetricCache from which any metrics can then be exported more easily